### ISSUE TITLE
Unable to use with Node.js 16.15

### ISSUE THREAD

**Issue by progsmile** [diamondio/better-queue-sqlite#8]
Hello dear developers! Thank you for the library!
I'm running the adapter from Node.js 16.15 and such error appears:
```
Uncaught Exception:
Error: The module '/home/denys_klymenko/Programming/mil/file-messenger/apps/file-messenger/.webpack/main/native_modules/lib/binding/node-v93-linux-x64/node_sqlite3.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 93. This version of Node.js requires
NODE_MODULE_VERSION 107. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
```
I see it might happen because of dependency `"sqlite3": "4.0.7"` that is quite old and supports only Node.js v12.x. Possible solution might be bumping sqilte3 to `5.1.2`
Could you assist with this please?
Thank you!

**Comment by leanderlee** [comment]
I just updated to sqlite@5.1.2 and using uuid@9, hopefully that fixes your problem. Published to version 1.0.7

**Comment by progsmile** [comment]
@leanderlee awesome, thank you!
